lib/modules/plugin/DeviceManagerEngine.ts

```diff
@@ -654,4 +653,40 @@ export class DeviceManagerEngine extends AbstractEngine<DeviceManagerPlugin> {
 
     return result.hits.map((elt) => elt._source);
   }
+
+  /**
+   * Detach all devices of an index in the admin index
+   *
+   * @param engineId The target engine Id
+   * @returns {any}
+   */
+  private async detachDevicesFromAdminIndex(engineId: string) {
```

Comment:
I feel like this method should be named `detachDevicesFromTenantIndex`

Reply:
I think tenantIndex could lead to confusion as this functions is related only to the 'platform' index which is referred as adminIndex in KDM.